src/SourceGenerators/Uno.UI.SourceGenerators/TSBindings/TSBindingsGenerator.cs

```diff
@@ -173,6 +195,24 @@ private bool IsMarshalledExplicitly(IFieldSymbol fieldSymbol)
 			throw new InvalidOperationException($"Failed to IsMarshalledExplicitly, unknown roslyn internal structure");
 		}
 
+		/// <summary>
+		/// Reads the actual symbol as Roslyn 3.6+ wraps symbols and we need access to the original type properties.
+		/// </summary>
+		/// <param name="symbol"></param>
+		/// <returns></returns>
+		private object GetActualSymbol(ISymbol symbol)
+		{
+			if (symbol.GetType().GetProperty("UnderlyingSymbol", System.Reflection.BindingFlags.Instance | System.Reflection.BindingFlags.NonPublic) is PropertyInfo info)
```

Comment:
Codacy found an issue: [Make sure that this accessibility bypass is safe here.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6270956)